src/components/ErrorPage/index.tsx

```diff
@@ -0,0 +1,43 @@
+import Icon from '../Icon';
+
+interface ErrorPageProps {
+  errorMessage: string;
+  text: string;
+  buttonContent: string;
+  onClick: () => void;
+}
+
+const ErrorPage = ({
+  errorMessage,
+  text,
+  onClick,
+  buttonContent,
+}: ErrorPageProps) => {
+  let errorText = 'ERROR';
+
+  if (errorMessage.includes('NetworkError')) {
+    errorText = 'NETWORK ERROR';
+  } else if (errorMessage.includes('404')) {
+    errorText = '404 ERROR';
+  } else if (errorMessage.includes('500')) {
+    errorText = 'SERVER ERROR';
+  }
```

Comment:
```suggestion
const errorMessage = {
  NetworkError : () => {errorText = NETWORK ERROR}
}
```
로직적인 차이가 발생하진 않지만, if -else로구현하면, 가독성도 떨어지고 시간복잡도면에서 미세하게나마 if-else를 순차로 탐색해야해서 조금 더 복잡할거에요 ! 

객체로 선언하는 방법도 있다 ~ 정도만 파악해주셔도 좋을 것 같습니다.